### ISSUE TITLE
Recent update did something to Webmin Sysstats

### ISSUE THREAD

**Issue by izeen** [qooob/authentic-theme#397]
Hi!

After the latest Authentic theme update the webmin sysstats subsite doesn't load. It just shows a blank page. Switching to any of the Webmin inbuilt themes makes the sysstats load correctly. Any chance of looking into this :) 

Sysstats module: http://webminstats.sourceforge.net

**Comment by qooob** [comment]
Hi,

Yes. I forgot to add an exclusion. It's very wrong that some modules not using `ui_print_footer();` functions. It's actually very wrong.

Here is the patch for CSS extensions in theme's settings:

```CSS
body.sysstats .container-fluid {
    pointer-events: auto !important;

    opacity: 1 !important;
}
```

Does it fix it? Yes? I will add it to the distro!

**Comment by izeen** [comment]
Thank you very much!

**Comment by qooob** [comment]
https://github.com/qooob/authentic-theme/issues/396

I'm creating exclusion list, stay tuned. It'll be in version 17.72.

**Comment by qooob** [comment]
Will be fixed in 17.72 and provide options for modules that need to be excluded.

https://github.com/qooob/authentic-theme/issues/396#issuecomment-185839541

**Comment by qooob** [comment]
Please upgrade to 17.72.
